gradle.properties

```diff
@@ -4,20 +4,20 @@ kotlin.code.style=official
 org.gradle.jvmargs=-Xmx3G
 # Fabric
 minecraft_version=1.16.4
-yarn_mappings=1.16.4+build.1:v2
-loader_version=0.9.3+build.207
-loom_version=0.5-SNAPSHOT
+yarn_mappings=1.16.4+build.9
+loader_version=0.11.1
+loom_version=0.6-SNAPSHOT
 # KAMI
 mod_version=dec
 maven_group=me.zeroeightsix
 archives_base_name=kami
 # Dependencies
-kotlin_version=1.4.0
+kotlin_version=1.4.21
 fiber_version=fix~undetected-mutation-SNAPSHOT
-kg_version=4e5eec4af9
-unsigned_version=1a71e303
-kool_version=4359096c
-glm_version=557908c3
-gli_version=4c290e0d
-gln_version=2c26a366
-uno_version=2599943c
+kg_version=7347a5d238
+unsigned_version=1e2fda82
+kool_version=b393e4c2
+glm_version=a4185eec
+gli_version=9c67885f
+gln_version=3e59f9ea
+uno_version=f60532b7
```

Comment:
Did you get these from the versions imgui uses, or are they just the latest commits? If it's the latter, I would prefer you get them from the imgui repo instead.

Mind that if this is a newer major version of imgui, some testing is due.

Reply:
kg_version is the latest imgui commit

the rest are the same from the last version bump in the kg/imgui gradle.properties.

I've tested it locally, but I still want to go through everything and make sure it's working properly as I would expect from imgui proper.

I will say I noticed a lot more tests being added to kg/imgui so I'm a bit more confident that it's not going to blow up.